client/my-sites/plugins/plugin-navbar-header/index.jsx

```diff
@@ -0,0 +1,662 @@
+import './style.scss';
+import { useTranslate } from 'i18n-calypso';
+
+const PluginNavbarHeader = () => {
+	const translate = useTranslate();
+
+	return (
+		<div>
+			<div className="x-root lpc-header-nav-wrapper">
```

Comment:
What's up with the classnames we're using here? Are we following the Calypso CSS coding standards?